Emby.Naming/Video/ExtraResolver.cs

```diff
@@ -79,6 +92,7 @@ private ExtraResult GetExtraInfo(string path, ExtraRule rule)
                     result.ExtraType = rule.ExtraType;
                     result.Rule = rule;
                 }
+                */
```

Comment:
The problem is that code is completely valid and could be reached when someone adds a new rule to NamingOptions.VideoExtraRules.
If someone finds a suitable regex to determine some extra they could just another VideoExtraRule that would not work. I think better option now that I look at this would be talor made test case to keep (and validate) the code.

Reply:
I have changed the test to check the regex instead of expecting the exception.